src/content/docs/apm/agents/net-agent/configuration/net-agent-configuration.mdx

```diff
@@ -3212,14 +3212,17 @@ There are three main sub-features:
 
 1. Metrics: Collect metrics on the total number of log lines written per harvest cycle (`Logging/lines`), as well as the number of log lines written at particular logging levels (for example, `Logging/lines/ERROR`).
 2. Log forwarding: When enabled, the agent will capture log data and send it to New Relic.
+  1. Context data (custom attributes): when enabled, the agent will capture and forward any custom log attributes.  The "include" and "exclude" elements are comma-separated lists of attribute names to include or exclude, following the [same rules](docs/apm/agents/net-agent/attributes/enable-disable-attributes-net/#attruls) as other agent attribute configuration.  They are both empty by default, which results in all log context data being captured and forwarded.
```

Comment:
The numbering looks wrong, but the preview is correct, so I'm not sure if it needs fixing